app/views/hyrax/transfers/index.html.erb

```diff
@@ -1,7 +1,13 @@
-<h1><%= t("hyrax.dashboard.transfer_of_ownership") %></h1>
+<% provide :page_header do %>
```

Comment:
See https://github.com/projecthydra-labs/hyrax/pull/706 for more information.  It's the way all the transfers to the template are done now in Hyrax